### PR TITLE
Translation documentation

### PR DESCRIPTION
**Pull request by emanlove**
I reworded some of the documentation from the perspective of one who had not yet used the
tool. Also made some minor changes to spelling.

### CODE REVIEW

**Review by aaltat — APPROVED** [review]

**Review by yuriverweij — APPROVED** [review]
found one more thing that could be changed (in 2 places), but looks good to me otherwise.

File: utest/test/api/approved_files/PluginDocumentation.test_many_plugins.approved.txt [hunk 1]
```diff
@@ -519,9 +519,10 @@ Template translation file, with English language can be created by running:
 languages, it only provides easy way to create full list keywords and their documentation in correct
 format. It is also possible to add keywords from library plugins by providing `--plugings` arguments
 to command. Example: `rfselib translation --plugings myplugin.SomePlugin /path/to/translation.json` The
-genered json file contains `sha256` key, which constains the sha256 sum of the library documentation,
-the sha256 sum is used by `rfselib translation --compare /path/to/translation.json` command, which compares
-transation to to library and prints outs a table which tell if there are changes needed for translation file.
+generated json file contains `sha256` key, which contains the sha256 sum of the library documentation.
+The sha256 sum is used by `rfselib translation --compare /path/to/translation.json` command, which compares
+the translation to the library and prints outs a table which tells if there are changes needed for
+translation file.
```
Comment (yuriverweij) [inline utest/test/api/approved_files/PluginDocumentation.test_many_plugins.approved.txt]: **the*** translation file

File: src/SeleniumLibrary/__init__.py [hunk 2]
```diff
@@ -580,9 +580,10 @@ class SeleniumLibrary(DynamicCore):
     languages, it only provides easy way to create full list keywords and their documentation in correct
     format. It is also possible to add keywords from library plugins by providing `--plugings` arguments
     to command. Example: `rfselib translation --plugings myplugin.SomePlugin /path/to/translation.json` The
-    genered json file contains `sha256` key, which constains the sha256 sum of the library documentation,
-    the sha256 sum is used by `rfselib translation --compare /path/to/translation.json` command, which compares
-    transation to to library and prints outs a table which tell if there are changes needed for translation file.
+    generated json file contains `sha256` key, which contains the sha256 sum of the library documentation.
+    The sha256 sum is used by `rfselib translation --compare /path/to/translation.json` command, which compares
+    the translation to the library and prints outs a table which tells if there are changes needed for
+    translation file.
```
Comment (yuriverweij) [inline src/SeleniumLibrary/__init__.py]: "... changes needed for **the** translation file."

File: utest/test/api/approved_files/PluginDocumentation.test_many_plugins.approved.txt (hunk 1 above)
Comment (emanlove) [reply utest/test/api/approved_files/PluginDocumentation.test_many_plugins.approved.txt]: Corrected. Thanks for this suggestion.

File: src/SeleniumLibrary/__init__.py (hunk 2 above)
Comment (emanlove) [reply src/SeleniumLibrary/__init__.py]: Also corrected

**Pull request merged**